modules/search/src/main/java/com/sygic/maps/module/search/SearchFragment.kt

```diff
@@ -79,7 +88,15 @@ class SearchFragment : Fragment(), SdkInitializationManager.Callback {
      *
      * @return [String] the text input value.
      */
-    var searchInput: String = "" // TODO: MS-5212
```

Comment:
chceme tu vobec mat public `searchInput`? Chapem ze mozes chciet nastavit pri vytvoreni text, ale to by mala byt jednorazova akcia pre konstrukcii a nie klasicky setter, ktory je mozne volat kedykolvek, ci?

Reply:
za mě to nevidím jen jako init hodnotu, ale možnost programově měnit vstup v search inputu. i kluci v iOS to tak mají.